samples/automl/automlNaturalLanguageModel.js

```diff
@@ -0,0 +1,599 @@
+  // [START automl_natural_language_getOperationStatus]
+  const automl = require(`@google-cloud/automl`);
+
+  const client = new automl.v1beta1.AutoMlClient();
+
+  /**
+   * TODO(developer): Uncomment the following line before running the sample.
+   */
+  // const operationFullId = `Full name of an operation, eg. “Projects/<projectId>/locations/us-central1/operations/<operationId>
+  // Get the latest state of a long-running operation.
+
+  // Get the latest state of a long-running operation.
+  client.operationsClient.getOperation(operationFullId).then(responses => {
+    const response = responses[0];
+    console.log(`Operation status: ${response}`);
+  });
+  // [END automl_natural_language_getOperationStatus]
+}
+
+function listModels(projectId, computeRegion, filter_) {
+  // [START automl_natural_language_listModels]
+  const automl = require(`@google-cloud/automl`);
+
+  const client = new automl.v1beta1.AutoMlClient();
+
+  /**
+   * TODO(developer): Uncomment the following line before running the sample.
+   */
+  // const projectId = `The GCLOUD_PROJECT string, e.g. "my-gcloud-project"`;
+  // const computeRegion = `region-name, e.g. "us-central1"`;
+  // const filter_ = `filter expressions, must specify field, e.g. “imageClassificationModelMetadata:*”`;
+
+  // A resource that represents Google Cloud Platform location.
+  const projectLocation = client.locationPath(projectId, computeRegion);
+
+  // List all the models available in the region by applying filter.
+  if (filter_ === ``) filter_ = `textClassificationModelMetadata:*`;
+  client
+    .listModels({
+      parent: projectLocation,
+      filter: filter_,
+    })
+    .then(responses => {
+      const model = responses[0];
+
+      // Display the model information.
+      console.log(`List of models:`);
+      for (let i of model) {
+        console.log(`Model name: ${i.name}`);
+        console.log(`Model id: ${i.name.split(`/`).pop(-1)}`);
+        console.log(`Model display name: ${i.displayName}`);
+        console.log(`Model dataset id: ${i.datasetId}`);
+        if (i.modelMetadata === `translationModelMetadata`) {
+          console.log(`Translation model metadata:`);
+          console.log(`\tBase model: ${i.translationModelMetadata.baseModel}`);
+          console.log(
+            `\tSource language code: ${
+              i.translationModelMetadata.sourceLanguageCode
+            }`
+          );
+          console.log(
+            `\tTarget language code: ${
+              i.translationModelMetadata.targetLanguageCode
+            }`
+          );
+        } else if (i.modelMetadata === `textClassificationModelMetadata`) {
+          console.log(
+            `Text classification model metadata: ${
+              i.textClassificationModelMetadata
+            }`
+          );
+        } else if (i.modelMetadata === `imageClassificationModelMetadata`) {
+          console.log(`Image classification model metadata:`);

```

Comment:
Nit: should we put this in a helper function, and include those helper functions with the sample?

Perhaps something like this?
```javascript
const _logImageClassificationModel = (model) => {
  ...
};
```
Apply this comment throughout **this file**.